code/controllers/subsystems/machinery.dm

```diff
@@ -69,6 +70,10 @@
 		if (rcon_update_queued)
 			build_rcon_lists()
 
+		if (powernet_update_queued)
+			makepowernets()
```

Comment:
Wouldn't it be an idea to swap all refs of `makepowernets` to use this boolean instead? There's roughly 2 more out there in the wild.